Userland/Libraries/LibWeb/Painting/GradientPainting.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Span.h>
+#include <AK/Vector.h>
+#include <LibGfx/Color.h>
+#include <LibWeb/CSS/StyleValue.h>
+#include <LibWeb/Forward.h>
+#include <LibWeb/Painting/PaintContext.h>
+
+namespace Web::Painting {
```

Comment:
please insert newline around namespaces

Reply:
:( clang-format needs to format harder :(

Reply:
yeah, that's one of the things it can't do unfortunately - one day it'll annoy me enough to send them a PR 😆